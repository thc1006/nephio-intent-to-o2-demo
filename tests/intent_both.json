{
  "intentId": "test-002",
  "serviceType": "massive-machine-type",
  "targetSite": "both",
  "resourceProfile": "standard",
  "sla": {
    "availability": 99.9,
    "connections": 1000000
  }
}