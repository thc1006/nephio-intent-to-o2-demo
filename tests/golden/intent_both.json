{
  "intentExpectationId": "golden-both-001",
  "intentExpectationType": "NetworkSliceIntent",
  "targetSite": "both",
  "intent": {
    "serviceName": "golden-service-multisite",
    "serviceType": "mMTC",
    "networkSlice": {
      "sliceId": "golden-slice-both-001",
      "sliceType": "mMTC",
      "maxNumberofUEs": 100000,
      "maxNumberofConnections": 500000
    },
    "qos": {
      "downlinkThroughput": "100Mbps",
      "uplinkThroughput": "50Mbps",
      "latency": "100ms",
      "reliability": "99%",
      "connectionDensity": "1000000/km2"
    },
    "coverageArea": {
      "site": "multi-site",
      "region": "us-all",
      "zones": ["zone-1", "zone-2"]
    },
    "lifecycle": {
      "startTime": "2025-01-01T00:00:00Z",
      "endTime": "2025-12-31T23:59:59Z"
    },
    "multiSite": {
      "loadBalancing": "round-robin",
      "failover": "automatic",
      "syncInterval": "30s"
    }
  }
}